### ISSUE TITLE
InfiniteScroll: true breaks ResourceHistogram

### ISSUE THREAD

**Issue by brianhjelle** [bryntum/support#5454]
I get an error when using `infiniteScroll: true` on ResourceHistogram. It seems like the error happens when the "resources" property is defined.

https://codepen.io/brianhjelle/pen/bGMXKeZ

```
schedulerpro.umd.js:53 Uncaught TypeError: Cannot read properties of undefined (reading 'isFiltered')
    at g.layoutResourceEvents (schedulerpro.umd.js:53:944448)
```

**Comment by matsbryntse** [comment]
Code pen test case looks incomplete, I don't see resources anywhere? 

I tried adding infiniteScroll in the code editor here and it works fine: https://www.bryntum.com/examples/gantt/resourcehistogram/

How can we reproduce?

**Comment by brianhjelle** [comment]
Sorry about that. I fixed the codepen. 
I can reproduce on the on the Bryntum website example also if you set `infiniteScroll: true` and add the following
```
resources: [
    { id: 1, name: "Arnold Smith", image: "arnold.jpg", calendar: 'workhours' },
    { id: 2, name: "Gloria Rogers", image: "gloria.jpg", calendar: 'workhours' },
    { id: 3, name: "Jane Miller", eventColor: "indigo", image: false,  calendar: 'workhours' }
  ],

  events: [
    {
      id: 1,
      name: "Interview",
      startDate: "2018-05-07",
      endDate: "2018-05-10",
    },
     {
      id: 2,
      name: "Interview",
      startDate: "2018-05-08",
      endDate: "2018-05-10",
    },
    { id: 3, name: "Audition", startDate: "2018-05-07", endDate: "2018-05-08" },
    {
      id: 4,
      name: "Script deadline",
      startDate: "2018-05-07",
      endDate: "2018-05-08"
    },
    { id: 5, name: "Rehearsal", startDate: "2018-05-09", endDate: "2018-05-11" }
  ],

  assignments: [
    { id: 1, eventId: 1, resourceId: 1, units: 33 },
    { id: 2, eventId: 2, resourceId: 1, units: 50 }, // 2 day event, 8 hour
    { id: 3, eventId: 3, resourceId: 2, units: 150 }, // 1 day, 12 hour
    { id: 4, eventId: 4, resourceId: 2 }, // 1 day, 8 hour
    { id: 5, eventId: 5, resourceId: 3 } // 2 day, 16 hours
  ]
```

**Comment by matsbryntse** [comment]
Thanks, reproduced1

**Comment by brianhjelle** [comment]
Hello, this is still an issue for my team. Do we have any update on this one?

**Comment by pwmather** [comment]
Hey @matsbryntse ,
We're also facing this problem too. Do you know when we can expect a fix?

**Comment by matsbryntse** [comment]
We'll try to fix it for the next patch or two, sorry for the delay!

**Comment by arcady-zherdev** [comment]
@pwmather FYI the upcoming v5.4.0 release (on `beta-1` stage now) includes refactored resource histogram and utilization views. I've just tried and couldn't reproduce this issue on there.
Could you try it yourself to make sure we are on the same page?

You can check it either online here: https://bryntum.com/products/schedulerpro-next/examples/resourcehistogram/
Or you can download the `5.4.0-beta-1` version to your local environment using `npm` or the customerzone (https://customerzone.bryntum.com/).

**Comment by pwmather** [comment]
thanks @arcady-zherdev - we will try it and report back.

**Comment by pwmather** [comment]
Hi @arcady-zherdev I can confirm that the autoHeight issue is resolved for us on 5.4.0-beta-1 version. Using this version we did see other issues with Gantt start and end dates disappearing but that a separate issue and hopefully resolved before release. If our dev team can create a simple test case that repro's I will raise that bug.
